src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -318,7 +325,31 @@ void onRTPTranslatorWillWriteAudio(
      * @param sz the number of black key frame packets to create.
      * @return the array of black key frame packets that were created.
      */
-    private static RawPacket[] make(int ssrc, int seqNumOff, long tsOff, int sz)
+    private RawPacket[] make(int ssrc, int seqNumOff, long tsOff, int sz)
```

Comment:
lol, sorry!  when i saw it pulling from the stream i just assumed it was reading from an incoming packet.  just ignore me then :)